test/cli_container_prune_test.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"strings"
+
+	"github.com/alibaba/pouch/test/command"
+	"github.com/alibaba/pouch/test/environment"
+
+	"github.com/go-check/check"
+	"github.com/gotestyourself/gotestyourself/icmd"
+)
+
+// PouchContainerPruneSuite is the test suite for container prune.
+type PouchContainerPruneSuite struct{}
+
+func init() {
+	check.Suite(&PouchContainerPruneSuite{})
+}
+
+// SetUpTest does common setup in the beginning of each test.
+func (suite *PouchContainerPruneSuite) SetUpTest(c *check.C) {
+	SkipIfFalse(c, environment.IsLinux)
+	SkipIfFalse(c, environment.IsCRIUExist)
+
+	PullImage(c, busyboxImage)
+}
+
+// TestContainerPruneWork tests "pouch container prune" work.
+func (suite *PouchContainerPruneSuite) TestContainerPruneWork(c *check.C) {
+	testContainer := command.PouchRun("create", busyboxImage125, "top").Assert(c, icmd.Success)
+	testContainerID := strings.TrimSpace(testContainer.Combined())
+
+	command.PouchRun("container", "prune", "-f").Assert(c, icmd.Success)
+
+	res := command.PouchRun("ps", "-a").Assert(c, icmd.Success)
+	if out := res.Combined(); strings.Contains(out, testContainerID) {
```

Comment:
we can reuse the logic with `util.PartialEqual`

Reply:
yes